src/index.html

```diff
@@ -4,24 +4,363 @@
       name="viewport"
-      content="width=device-width, initial-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
     />
     <title>Catalog</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
     <link
       rel="preconnect"
       href="https://fonts.gstatic.com"
+      crossorigin=" "
     />
     <link
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
     />
+
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.css"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__logo"
+        href="index.html"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="is-active nav__link nav__link--1"
+              href="index.html"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="cards_container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="image"
+        />
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
```

Comment:
This tag used for title of the page, so you need to use the other tag here
```suggestion
        <h2 class="card__title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h2>
```